sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -157,7 +155,8 @@ public void testSelectCountStart() throws Exception
         CalciteTests.REGULAR_USER_AUTH_RESULT,
         ImmutableList.of(Druids.newTimeseriesQueryBuilder()
                                .dataSource(CalciteTests.DATASOURCE1)
-                               .intervals(querySegmentSpec(Intervals.of("2999-01-01T00:00:00.000Z/146140482-04-24T15:36:27.903Z")))
+                               .intervals(querySegmentSpec(Intervals.of(
```

Comment:
formatting here also doesn't fit style guide, closing parens should be on own line

Reply:
Weird, Intellij's autoformatter with the Druid style doesn't pick that up.